vagrant/provisioning/roles/elasticsearch-install/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+    # tasks file for /etc/ansible/roles/elastic
+    - name: Copy the ElasticSearch repo
+      template:
+        src: elasticsearch.repo
+        dest: /etc/yum.repos.d/
+        backup: yes
+    
+    - name: Import GPG key for ElasticSearch   
+      rpm_key:
+        state: present
+        key: https://artifacts.elastic.co/GPG-KEY-elasticsearch      
+
+    - name: Install ElasticSearch
+      yum:
+        name: elasticsearch
+        state: present
+
+    - name: Elasticsearch folders
+      become: yes
+      file:
+        path: "{{ item }}"
+        state: directory
+        group: elasticsearch
+        owner: elasticsearch
+      loop:
+        - "{{ root_folder }}/data/elasticsearch"
+        - "{{ root_folder }}/log/elasticsearch"
+
+    - name: Copying ElasticSearch configuration files
+      template:
+        src: elasticsearch.yml
+        dest: /etc/elasticsearch
+        backup: yes
+    
+    - name: Adding services to auto start
+      systemd:
+        name: elasticsearch
+        enabled: yes
+        daemon_reload: yes
+        masked: no
+        state: started
+      when: enable_elasticsearch is defined and enable_elasticsearch == "yes"
+ 
+    - name: Wait for 30 seconds for elasticsearch to become responsive...
```

Comment:
Here consider waiting for the elasticsearch webapp to be responsive, like with Bojan's comment here, https://github.com/ArkCase/arkcase-ce/pull/75

Reply:
@david-oc-miller , in my last commit I included the 'uri' ansible module with the main role of waiting for elasticsearch to be responsive. In a consultation with Dimitar, as a variable for remote host where elasticsearch will be up and running, i use 'external_host'.